woodwork/type_sys/type_system.py

```diff
@@ -93,7 +93,7 @@
 
 DEFAULT_TYPE = Unknown
 
-INFERENCE_SAMPLE_SIZE = 100000
+INFERENCE_SAMPLE_SIZE = 100_000
```

Comment:
We might have to leave this as 100,000. I'm seeing errors popping up in larger datasets like `zillow.csv` where `IntegerNullable` is inferred from 10,000 but missing values exist in the 80,000 remaining observations. This throws an error during initialization since `Int64` isn't compatible with `NaNs`.